### PR TITLE
feat: add exporter

### PR DESCRIPTION
**Pull request by dejanb**

### CODE REVIEW

File: chart/dev.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  indexedTopic: sbom-indexed-dev
+  failedTopic: sbom-failed-dev
+  exporterTopic: sbom-guac-dev
+vexination:
+  bucket: vexination-dev
+  credentials: vexination-credentials-secret
+  storedTopic: vex-stored-dev
+  indexedTopic: vex-indexed-dev
+  failedTopic: vex-failed-dev
+  source: https://www.redhat.com/.well-known/csaf/provider-metadata.json
```
Comment (lulf) [inline chart/dev.yaml:20]: Should probably use the same source as staging

**Review by lulf — COMMENTED** [review]
I think the exporter should be under bombastic/exporter, and named/labeled appropriately:

```
    app.kubernetes.io/name: bombastic-exporter
    app.kubernetes.io/component: exporter
    app.kubernetes.io/part-of: bombastic
```

When we have vex exporting working, another template should be added for that, even if they're using the same 'binary' they will have different arguments.

**Comment by dejanb** [comment]
@lulf Fixed all the suggestions. Thanks

**Review by lulf — CHANGES_REQUESTED** [review]
Looks great, some final nitpicking (sorry)

**Review by lulf — APPROVED** [review]

**Pull request merged**